pkgs/by-name/mi/microfetch/package.nix

```diff
@@ -6,22 +6,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "microfetch";
-  version = "0.3.3";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = "NotAShelf";
     repo = "microfetch";
     rev = "refs/tags/v${version}";
-    hash = "sha256-iBfnui7xrx31euYnQxoJf1xXFYFiJnDMadLRAiRCN04=";
+    hash = "sha256-MWb768ipDgFnnVGxtk754w4SDomGgPItye/k18EHAZM=";
   };
 
-  cargoHash = "sha256-J+S6XIMUcWj4jXABQinEDx6dRG5Byc2UxJoKj2y1tQU=";
+  cargoHash = "sha256-bu3fX6tTJLHjMbwYvoCqqAvKONmtuMaV0PdvrZf5WbY=";
 
   meta = {
     description = "Microscopic fetch script in Rust, for NixOS systems";
     homepage = "https://github.com/NotAShelf/microfetch";
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [ nydragon ];
```

Comment:
Let's not remove the maintainers attribute in the version update commit

Reply:
This should've been fixed now!